recipes/pycln/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "pycln" %}
+{% set version = "2.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pycln-{{ version }}.tar.gz
+  sha256: aeba833e233c8bb0c4945d4b61ecbcb08a0441c89d8f018d0ee904e7bba8efab
+
+build:
+  skip: true  # [py>=400]
```

This package is pure Python and can be noarch:
```suggestion
  noarch: python
```